### PR TITLE
Update dns pod label modification

### PR DESCRIPTION
**Pull request by techcet**
Update command to edit kube-dns labelled pods.

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/5634)

### CODE REVIEW

**Comment by ianvernon** [comment]
@techcet please sign off on your commit, c.f. [the Developer's Certificate of Origin](http://docs.cilium.io/en/v1.2/contributing/#developer-s-certificate-of-origin)

File: examples/kubernetes/addons/etcd-operator/README.md [hunk 1]
```diff
@@ -54,12 +54,13 @@ tls/deploy-certs.sh
...
 ------------------------------------------------------------
 
-Before running this step, make sure you have `kube-dns` running and contains
-the label `io.cilium.fixed-identity=kube-dns`.
+Ensure your DNS service is running and contains the label `io.cilium.fixed-identity=kube-dns` using the following command.
 
-```
-kubectl label -n kube-system pod $(kubectl -n kube-system get pods -l k8s-app=kube-dns -o jsonpath='{range .items[]}{.metadata.name}{" "}{end}') io.cilium.fixed-identity=kube-dns
-```
+# if using kube-dns
```
Comment (ianvernon) [inline examples/kubernetes/addons/etcd-operator/README.md:59]: Add three backticks around each `kubectl patch` command, e.g. \`\`\` <command> \`\`\`

**Review by ianvernon — CHANGES_REQUESTED** [review]

**Review by aanm — APPROVED** [review]

File: examples/kubernetes/addons/etcd-operator/README.md (hunk 1 above)
Comment (techcet) [reply examples/kubernetes/addons/etcd-operator/README.md:59]: Done

**Comment by techcet** [comment]
@ianvernon I suspect I may have made edits directly on the GH webpage, and missed the sign-off that I usually do with my normal workflow. How does it look now?

**Review by ianvernon — APPROVED** [review]

**Comment by ianvernon** [comment]
@techcet LGTM, merging. No need to run CI since this is just a README change.

**Comment by techcet** [comment]
Request to backport for 1.2.3 (Ubuntu 18.04 blog references this README).

**Pull request merged**